assets/section-product-recommendations.css

```diff
@@ -6,3 +6,14 @@
   margin: 0;
   margin-bottom: 3rem;
 }
+
+.product-recommendations:not(.product-recommendations--loaded) {
```

Comment:
I'm not sure this addresses the background color of the container? Could we simply set:
``` css
.product-recommendations:not(.product-recommendations--loaded) {
  display: none;
}
```

![Screen Shot 2022-01-10 at 3 19 01 PM 2](https://user-images.githubusercontent.com/3710548/148833624-72bdc07d-1097-4289-a4c7-9867c9741f6f.png)

Reply:
This is a good idea - I didn't think of just doing that 🤦 Much simpler!